src/components/protocol_handler/include/protocol_handler/protocol_handler_impl.h

```diff
@@ -389,6 +393,7 @@ class ProtocolHandlerImpl
    * Only valid when generated_session_id is 0. Note, even if
    * generated_session_id is 0, the list may be empty.
    */
+  // DEPRECATED
   void NotifySessionStartedResult(
```

Comment:
Merge current develop branch, so that the DEPRECATED macro can be used.

Reply:
@jacobkeeler all commented DEPRECATED keywords in pull request were replaced to macro